pkg/apis/velero/v1/delete_backup_request_types.go

```diff
@@ -58,6 +58,8 @@ type DeleteBackupRequestStatus struct {
 // +kubebuilder:storageversion
 // +kubebuilder:printcolumn:name="BackupName",type="string",JSONPath=".spec.backupName",description="The name of the backup to be deleted"
 // +kubebuilder:printcolumn:name="Status",type="string",JSONPath=".status.phase",description="The status of the deletion request"
+// +kubebuilder:rbac:groups=velero.io,resources=deletebackuprequests,verbs=get;list;watch;create;update;patch;delete
```

Comment:
IMHO, the RBAC-related markers should be put in the controller code, this is the best practice because it's in the controller code where we operate the CRs?  Correct me if I'm wrong

Reply:
The reason for moving to type code is many controllers all need access to the backup resource, and they have different permission settings for that. That makes the controller-gen CLI confused.
Before this change, the generated backup's permission is not correct in the role.yaml file.

I think we don't need to clarify the exact resources it needs in the controllers. Since the CRD resources are owned by Velero, just ask for full permission once makes sense. And it's possible that some CRDs don't have related controllers.

I also agree that there are some scenarios in that controllers need non velero resource permissions. I put the PVB's pod, PV and PVC RBAC setting to type files too.
It's not easy to cover. For a functional installation, Velero needs permissions for all resources in the cluster, so this is just an example of RBAC setting for Velero-owned resources.